flows/vision-zero/db_permission_grantor/db_permission_grantor.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+
+import os
+import json
+
+import prefect
+from prefect.backend import get_key_value
+from prefect import Flow, task
+from prefect.run_configs import UniversalRun
+import psycopg2
+
+kv_store = get_key_value("Vision Zero")
+kv_dictionary = json.loads(kv_store)
+
+DB_HOST = None
+DB_USER = None
+DB_PASS = None
+DB_NAME = None
+SCHEMAS_CDL = None
+STAFF_CDL = None
+
+if False:
+    DB_HOST = os.getenv("DB_HOST")
+    DB_USER = os.getenv("DB_USER")
+    DB_PASS = os.getenv("DB_PASS")
+    DB_NAME = os.getenv("DB_NAME")
+    SCHEMAS_CDL = os.getenv("DB_PERMISSION_SCHEMAS")
+    STAFF_CDL = os.getenv("DB_PERMISSION_STAFF")
+else:
+    DB_HOST = kv_dictionary["DB_HOST"]
+    DB_USER = kv_dictionary["DB_USER"]
+    DB_PASS = kv_dictionary["DB_PASS"]
+    DB_NAME = kv_dictionary["DB_NAME"]
+    SCHEMAS_CDL = kv_dictionary["DB_PERMISSION_SCHEMAS"]
+    STAFF_CDL = kv_dictionary["DB_PERMISSION_STAFF"]
```

Comment:
@frankhereford would it be possible to fetch the list of staff users through a DB query instead of having to maintain the list of staff in the kv store?

Reply:
@johnclary In a way, yes, but we'd still need to keep track of a list of 'non-user' accounts, which may be better.

Here's an example of `SELECT rolname from pg_roles;`with other staff members names blurred out.
![roles](https://user-images.githubusercontent.com/3653206/209980970-cf575590-1300-493c-b2a8-6a15ff19c266.jpg)

We could hard-code in all the non-staff members and do some set operations to get at a dynamic list of staff members. I'm not sure if this really helps us all that much - maybe I'm not seeing the whole thing clearly.

Same, but to lesser degree, with the schemas - I think it's generally safer to have an explicit list of what we're targeting for our `GRANT`s, as we're giving privilege. I'd be more inclined to have a wider net if we were taking away access. 

I'm super open to discussion - what do you think?